### ISSUE TITLE
Flag for booshers

### ISSUE THREAD

**Issue by mkaito** [Eve-PySpy/PySpy#18]
Just like the flags for BLOPS and CYNO, I would like to see a flag for Booshers, triggered if the pilot is known to fly command destroyers. This would be useful for Rorqual pilots, for example.

**Comment by leonhess** [comment]
This would be nice to have. It requires some work in the backend but once that is done we easily could add Flags for all ship types.
But before that we would like to implement the asynchronous fetching for the frontend (#15 ) and that will probably still take a while since me and @Crypta-Eve are pretty busy atm.
